pennylane/tape/__init__.py

```diff
@@ -40,6 +41,128 @@
 _mock_stack = []
 
 
+class TapeOperationRecorder(QuantumTape):
+    """A template and quantum function inspector,
+    allowing easy introspection of operators that have been
+    applied without requiring a QNode.
+
+    **Example**:
+
+    The OperationRecorder is a context manager. Executing templates

```

Comment:
```suggestion
    The TapeOperationRecorder is a context manager. Executing templates
```

Reply:
The user will never see the name `TapeOperationRecorder` 🙂 

This will instead replace `qml._queueing.OperationRecorder` in the exact same location with the same name, when `qml.enable_tape()` is called.